src/Entity/Tank/Projectile/Bullet.ts

```diff
@@ -73,7 +73,7 @@ export default class Bullet extends LivingEntity {
         this.relations.values.team = barrel.relations.values.team;
         this.relations.values.owner = tank;
 
-        this.physics.values.sides = 1;
+        this.physics.values.sides = bulletDefinition.sides || 1;
```

Comment:
Maybe use `??` instead of `||` for the bulletDefinition.sides, that way people can set it to 0 (rare).